include/fe/fe.h

```diff
@@ -686,7 +690,9 @@ class FEHierarchic : public FE<Dim,HIERARCHIC>
    * to be used in dimension \p Dim.
    */
   explicit
-  FEHierarchic(const FEType& fet);
+  FEHierarchic(const FEType& fet) :
+    FE<Dim,HIERARCHIC> (fet)
+  {}
```

Comment:
Wait, never mind, that's not a catch, just a reorganization.  At some point we should nail down style guidelines about when inline functions should go inside the class definition and when outside.